redhat-developer/gitops-operator
# Add e2e tests for ArgoCD route controller

- e2e tests to check if the console link is being created
- Added scripts for e2e setup
- Added Makefile for common commands

#### How to test:
1. Login to your OpenShift cluster
2. Make sure operator-sdk is installed
3. Run `make test-e2e`
